r4/resource-apis/account/Package.md

```diff
@@ -0,0 +1,57 @@
+
+
+# Account Template
+
+## Template Overview
+
+This template provides a boilerplate code for rapid implementation of FHIR APIs and creating, accessing and manipulating FHIR resources.
+
+
+| Module/Element       | Version |
+|---| --- |
+| FHIR version         | r4 |
+| Implementation Guide | [http://hl7.org/fhir](http://hl7.org/fhir) |
+| Profile URL          |[http://hl7.org/fhir/StructureDefinition/Account](http://hl7.org/fhir/StructureDefinition/Account)|
+
+### Dependency List
+
+- ballerinax/health.fhir.r4
+- ballerinax/health.fhirr4
+- ballerinax/health.fhir.r4.international401
+
+This template includes,
+
+- Ballerina service for 'FHIR Resource Name' FHIR resource with following FHIR interactions.
+- READ
+- VREAD
+- SEARCH
+- CREATE
+- UPDATE
+- PATCH
+- DELETE
+- HISTORY-INSTANCE
+- HISTORY-TYPE
+- Generated Utility functions to handle context data
+
+## Prerequisites
+
+Pull the template from central
+
+    ` bal new -t ballerinax/health.fhir.templates.international401.account AccountAPI `
+
+## Run the template
+- Run the Ballerina project created by the service template by executing bal run from the root.
+- Once successfully executed, Listener will be started at port 9090. Then you need to invoke the service using the following curl command
+    ` $ curl http://localhost:9090/fhir/r4/Account `
+- Now service will be invoked and returns an Operation Outcome, until the code template is implemented completely.
+
+## Adding a Custom Profile/Combination of Profiles
+
+- Add profile type to the aggregated resource type. Eg: `public type Account r4:Account|<Other_Account_Profile>;`.
+    - Add the new profile URL in `api_config.bal` file.
+    - Add as a string inside the `profiles` array.
+    - Eg: `profiles: ["http://hl7.org/fhir/StructureDefinition/Account", "new_profile_url"]`
+
+## Onboard as a Choreo project
```

Comment:
We can't directly onboard as a choreo project. we need the openapi and the endpoints yaml.

Reply:
As discussed offline, for now the instructions related to on-boarding on Choreo are removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.templates/commit/823ce4003ce19803b81239a8fd2bc79c2f1803e8 until the openapi generation for fhir service is supported.